scripts/link-vms-by-sr.py

```diff
@@ -1,41 +1,37 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 # Populate a directory of symlinks partitioning VMs by SR
 # (c) Anil Madhavapeddy, Citrix Systems Inc, 2008
 
+from __future__ import print_function
+
+import argparse
 import atexit
+import contextlib
+import os
+import sys
+from pathlib import Path
+
 import XenAPI
-import os, sys
-import getopt
 
-def logout():
-    try:
+
+def logout(session):
+    with contextlib.suppress(Exception):
         session.xenapi.session.logout()
-    except:
-        pass
-atexit.register(logout)
-
-def usage():
```

Comment:
I don't see usage in your change. Did you remove it？

Reply:
ArgumentParser take care of it?